### PR TITLE
Add docs about build admin frontend

### PR DESCRIPTION
**Pull request by alexander-schranz**
| Q | A
| --- | ---
| Fixed tickets | fixes #issuenum
| Related PRs | https://github.com/sulu/skeleton/issues/68#issuecomment-795089277
| License | MIT

#### What's in this PR?

From: https://github.com/sulu/skeleton/issues/68#issuecomment-795089277

#### Why?

Having it in the docs should not hurt.

### CODE REVIEW

File: cookbook/build-admin-frontend.rst [hunk 1]
```diff
@@ -0,0 +1,74 @@
+Building Admin Frontend
+=======================
```
Comment (niklasnatter) [inline cookbook/build-admin-frontend.rst]:
```suggestion
Building the administration interface frontend application
==========================================================
```

File: cookbook/build-admin-frontend.rst [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+Solution 1: Update Command (Recommended way)
```
Comment (niklasnatter) [inline cookbook/build-admin-frontend.rst:9]:
```suggestion
The administration interface of Sulu is implemented as a single page application using React. The code of this 
application is built using webpack and stored in the `public/build/admin` directory of the project. 
If you update your Sulu version or add custom Javascript code to your project, you need to update the build in 
the `public/build/admin directory`. There are different ways to do this:

Solution 1: Update Command (Recommended way)
```

File: cookbook/build-admin-frontend.rst [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+3. Create the administration interface build
+
+.. code-block:: bash
+
+    cd /var/project/assets/admin
+    npm install
+    npm run build
+
+.. _node: https://nodejs.org/en/
```
Comment (niklasnatter) [inline cookbook/build-admin-frontend.rst:81]:
```suggestion
.. _sulu/skeleton repository: https://github.com/sulu/skeleton
.. _node: https://nodejs.org/en/
```

File: cookbook/build-admin-frontend.rst [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+
+.. code-block:: bash
+
+   bin/adminconsole sulu:admin:update-build
+
+.. note::
+
+   If a manually build is needed for example if you added custom
+   JavaScript to your Project. You need to make sure that you
+   have `node`_ installed to build it.
```
Comment (niklasnatter) [inline cookbook/build-admin-frontend.rst]:
```suggestion
   The update build command will download the build for your project from the `sulu/skeleton repository`_ if possible.
   If you have added custom Javascript code in your project, the command will automatically cleanup leftovers from
   previous builds and build the Javascript code manually on your system. Make sure that you have installed `node`_ if 
   your project requires a manual build.
```

**Comment by alexander-schranz** [comment]
@nnatter applied 👍

**Review by niklasnatter — APPROVED** [review]

**Pull request merged**